requirements-devel.txt

```diff
@@ -11,7 +11,7 @@ codespell==2.2.1
 coverage==6.5.0
 craft-cli==1.2.0
 craft-grammar==1.1.1
-craft-parts==1.15.0
+craft-parts==1.17.1
```

Comment:
this is 2 versions of bumping for hotfix, what else are we taking in that could be risky?

Reply:
There are many changes there, we can backport the git scheme change to 1.15 series and use that instead.

Reply:
PR #3993 created to update the 7.2 hotfix branch to craft-parts 1.15.2. This PR retargeted to main to update it with the corresponding change.